iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -138,7 +144,7 @@ public TimeSpan OpenTimeout
         }
 
         /// <summary>
-        /// The pre-fetch count
+        /// The number of messages that the message receiver can simultaneously request. The default is 50.
```

Comment:
This is what the official meaning is but I was not able to use it or figure out what this does in reality. Anyone has any info?

Reply:
iirc this was being corelated to the no of c2d messages a client queue could hold and receive at once. Since the c2d message queue cannot be greater than 50, I am surprised that this is user-configurable (maybe can be lowered but not increased beyond 50).

Reply:
Ok in that case let me check with the service team before updating the comment.

Reply:
I have not heard back from the service team yet. So I'll leave the statement as it is and update in a subsequent PR.